docs-src/python/how-to-set-a-parent-close-policy-in-python.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-set-a-parent-close-policy-in-python
+title: How to set a Parent Close Policy in Python
+sidebar_label: Parent Close Policy
+description: Create an instance of the `ParentClosePolicy` class.
+tags:
+  - python
+  - developer-guide
+  - how-to
+---
+
+Create an instance of the [`ParentClosePolicy`](https://python.temporal.io/temporalio.workflow.ParentClosePolicy.html) class and specify a constant to determine how a Child Workflow should be handled when the Parent closes.
```

Comment:
You don't create an instance `ParentClosePolicy`, you just set the `parent_close_policy` parameter.

Not sure code so simple as "set parameter X" in our SDKs really needs its own section